zerver/actions/create_realm.py

```diff
@@ -214,6 +215,7 @@ def do_create_realm(
         kwargs["enable_read_receipts"] = (
             invite_required is None or invite_required is True or emails_restricted_to_domains
         )
+    kwargs["push_notifications_enabled"] = sends_notifications_directly()
```

Comment:
This might deserve a comment of the form `# Initialize this setting correctly in the case that no network activity is required to do so correctly."